cmd/expandybird/expander/expander_test.go

```diff
@@ -756,44 +756,16 @@ func TestReplicatedService(t *testing.T) {
 		&expansion.ServiceResponse{
 			Resources: []interface{}{
 				map[string]interface{}{
```

Comment:
Someone may have made some sort of library to give a nice structural diff of this kind of data.  That would make it much easier to interpret test failures.